python/amici/ode_export.py

```diff
@@ -247,49 +252,46 @@
 logger = get_logger(__name__, logging.ERROR)
 
 
-
-def var_in_function_signature(name, varname):
-    """Checks if the values for a symbolic variable is passed in the signature
+def var_in_function_signature(name: str, varname: str) -> bool:
+    """
+    Checks if the values for a symbolic variable is passed in the signature
     of a function
 
-    Arguments:
-
-        name: name of the function @type str
-
-        varname: name of the symbolic variable @type str
-
-    Returns:
-
-    Raises:
+    :param name:
```

Comment:
I think I liked the old comment formatting better for reading in-source. :grimacing:

Reply:
Thats the format that swig generates though and the formatting that sphinx generates is different for these formats

Reply:
Also pyment can interconvert docstring formats, so switching to a different supported format such as numpydoc shouldn’t be such a big problem.

Reply:
Agreeing that it is preferable to have a consistent style. This one is also better supported by e.g. pycharm. Also RST allows for some other nice features. Still finding it rather ugly :). Wasn't aware, that sphinx produces different results of it. Used the old style also e.g. in PEtab, and things looked well after sphinx. Never compared.

Reply:
Could also have been me not properly setting up sphinx at first :D
Lets see what things look like when this is done, I think most of the refactoring should be done :)